vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/GridMultiSelectionModel.java

```diff
@@ -93,7 +97,12 @@ Registration addMultiSelectionListener(
      * <p>
      * The default value is {@link SelectAllCheckboxVisibility#DEFAULT}, which
      * means that the checkbox is only visible if the grid's data provider is
-     * in- memory.
+     * in-memory.
```

Comment:
Again, I would now rephrase to not speak about data providers as most will not be using those anymore (?)

Reply:
Done